src/main/java/com/minecolonies/coremod/entity/ai/citizen/planter/EntityAIWorkPlanter.java

```diff
@@ -199,7 +199,7 @@ protected IAIState decide()
                 return PLANTATION_PLANT;
             }
         }
-
+        
```

Comment:
Why'd you add this whitespace?

Reply:
Think it was accidentally left-over from fighting with PLANT2.  Will see about fixing that next commit.

Reply:
Probably accidental when working on the PLANT_2 research effect.  Thanks for catching that.

Reply:
Ofc, no problem